src/main/resources/templates/default/require_exact/.github/workflows/release_droid_prepare_original_checksum.yml

```diff
@@ -0,0 +1,32 @@
+name: Release Droid - Prepare Original Checksum
```

Comment:
I'm not sure how it works here,  but why do you only have `Prepare Original Checksum` workflow in this folder? Where is `Print Quick Checksum`?